pkg/trace/config/config.go

```diff
@@ -160,6 +161,7 @@ func New() *AgentConfig {
 
 		ExtraSampleRate: 1.0,
 		TargetTPS:       10,
+		ErrorTPS:        10,
```

Comment:
One thing I'm concerned about is that this change is not backwards compatible. Previously, the ErrorTPS was whatever values TargetTPS was. Should we not keep that logic? I mean:
* If `ErrorTPS` is not set, use the value of `TargetTPS`
* If `ErrorTPS` is set, use that
I think this would be done in `apply.go`

Reply:
What do you think of adding a different targetTPS config to allow configuring head base tps without setting error tps?